src/effects/RealtimeEffectList.cpp

```diff
@@ -25,7 +26,7 @@ std::unique_ptr<ClientData::Cloneable<>> RealtimeEffectList::Clone() const
 {
    auto result = std::make_unique<RealtimeEffectList>();
    for (auto &pState : mStates)
-      result->mStates.push_back(RealtimeEffectState::make_shared(*pState));
+      result->mStates.push_back(pState);
```

Comment:
This change is new since I last reviewed, and I would describe this oppositely -- what was a deep copy is now shallow.

This has the consequence that the state is only transient state.  There is no longer any undoing or redoin of changes of settings, but only of the composition of the effect list.

Reply:
Yeah, you are right, my bad